### DISCUSSION TITLE
如何允许TimePicker跨天选择 (开始小时大于结束小时)

### DISCUSSION THREAD

**Discussion by Statfine** [ant-design/ant-design#41700] [Q&A] [answered]
当我开始时间选择12:00结束时间选择11:00，其实是希望当日的12点到日次的11点，但是组件内部就帮你转换了[12:00, 11:00] TO [11:00,12:00]，但其实我只想保留我的选择[12:00, 11:00]， 是否有属性支持开始时间(小时)大于结束时间(小时)

**Comment by Wxh16144** [comment, marked as answer]
Duplicate of #41632
